build/root/WORKSPACE

```diff
@@ -5,16 +5,16 @@ load("//build:workspace_mirror.bzl", "mirror")
 
 http_archive(
     name = "io_bazel_rules_go",
```

Comment:
FYI ideally we switch to consuming this from repo-infra

Reply:
@fejta -- Thanks. Rebased to pull in your repo-infra@0.0.4 bump.

Reply:
I'd expect to see this line change to reference 1.14.2?

https://github.com/kubernetes/kubernetes/blob/501781abe4290e19e9b42d292b0b295ea129dea6/build/root/WORKSPACE#L21-L22

Reply:
@fejta -- Ahhhhh, good catch! That got missed in the rebase since the lines moved. Will fix up.